commands.js

```diff
@@ -817,6 +826,40 @@ function regCommands(commandManager) {
                     utils.bot.sendChat('@' + user.username + ' you got moved to spot #' + spot + ' in queue.');
                 });
             }
+        ),
+        new Command('setrdj', ['setrdj', 'set_rdj'], 0, ['mod'], [],
+            /**
+             * @param {MessageUtils} 
+            */
+            function (utils) {
+                var user = utils.getTargetName(1, true);
+                if (!user) {
+                    return;
+                }
+                var username = user;
+                if (!(user = utils.bot.getUserByName(username, true))) {
+                    utils.bot.sendChat('@' + utils.getUserUsername() + ' ' + username + ' is not online.');
+                    return;
+                } 
+                var rdjrequireddubs = 10000;
```

Comment:
This would be better as an environment variable instead of being hardcoded in here.